app/models/block.rb

```diff
@@ -29,15 +29,15 @@ class Block < ApplicationRecord
 
   validate :not_blocking_admin
   def not_blocking_admin
-    errors.add(:blocked, "cannot be an admin") if blocked.has_role?(:admin)
-    errors.add(:blocked, "cannot be a moderator") if blocked.title == 'Mod'
+    errors.add(:blocked, 'cannot be an admin') if blocked.has_role?(:admin)
+    errors.add(:blocked, 'cannot be a moderator') if blocked.title == 'Mod'
   end
 
   def self.hidden_for(user)
     return [] if user.nil?
     user = user.id if user.respond_to?(:id)
-    Block.where('user_id = ? or blocked_id = ?', *[user]*2).
-          pluck(:blocked_id, :user_id).flatten.uniq - [user]
+    Block.where('user_id = ? or blocked_id = ?', *[user] * 2)
```

Comment:
prolly actually best to just do `.where('user_id = ? or blocked_id = ?', user, user)` instead of my stupid cleverness

Reply:
(I was trying to figure out the meaning behind this, thanks for explaining)

Reply:
Yeah it builds an array of two `user` (multiplies `[user]` times `2`) and then uses the splat operator (`*`) on it to make those into separate parameters.  It's clever, but it's also really really hard to read and I should never have done it

Reply:
I mean I'll give you credit the idea is A+++ but other than that...